### DISCUSSION TITLE
[4.0] Order Condition in first place in Search Tools

### DISCUSSION THREAD

**Discussion by coolcat-creations** [joomla/joomla-cms#47171] [2. Cold Features]
### What needs to be fixed
<img width="1142" alt="grafik" src="https://user-images.githubusercontent.com/828371/79065196-b6b9b900-7cae-11ea-8a99-9b740a2d7d1e.png">
 The Condition of the Article is ordered on 3rd place in  the Search tools. It's probably one of the most used dropdowns and user learned in all Joomla versions to find it at position one. I would recommend to order it at the first place in the search tools.

**Comment by brianteeman** [comment]
The filters are ordered to match the order of the data

**Comment by coolcat-creations** [comment]
I understand, I am not sure if the user will be happy with the conditions to be changed somewhere in between. What do you think?

**Comment by infograf768** [comment]
I agree with @coolcat-creations 
Condition should be first. 
Stage: no idea...
Featured after both

**Comment by chmst** [comment]
I agree. 

I don't know whether I ever used the featured filter or access in my pages and it would be a great improvement if a user could switch off the superfluous filters.  

Btut he whole search filter is a work in prograss at the moment, as Angie Radtkes concept could change it all an a11y requirements still are open. So at the we better wait for this.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28664">issues.joomla.org/tracker/joomla-cms/28664</a>.</sub>

**Comment by brianteeman** [comment]
@chmst there have been no significant changes to the filters so I guess a decision can be made now.

As I stated above the filters are currently in the same order as the columns. The same is true for the ordering select dropdown.

If you want to change this so that status is always first then I would suggest that it is done in both places and for all components.
